addons/l10n_mx/models/chart_template.py

```diff
@@ -39,3 +39,10 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
             'show_on_dashboard': True,
         })
         return res
+
+    def _get_account_vals(self, company, account_template, code_acc, tax_template_ref):
+        val = super(AccountChartTemplate, self)._get_account_vals(
+            company, account_template, code_acc, tax_template_ref)
+        val.update({'group_id': (account_template.group_id.id if
```

Comment:
Could you try with only

val.update({'group_id': account_template.group_id.id})